stable_baselines3/common/vec_env/vec_video_recorder.py

```diff
@@ -1,13 +1,180 @@
+import json
 import os
-from typing import Callable
+import os.path
+import tempfile
+from typing import Callable, List, Optional
 
-from gymnasium.wrappers.monitoring import video_recorder
+import numpy as np
+from gymnasium import error, logger
 
 from stable_baselines3.common.vec_env.base_vec_env import VecEnv, VecEnvObs, VecEnvStepReturn, VecEnvWrapper
 from stable_baselines3.common.vec_env.dummy_vec_env import DummyVecEnv
 from stable_baselines3.common.vec_env.subproc_vec_env import SubprocVecEnv
 
 
+# This is copy and pasted from Gymnasium v0.26.1
+class VideoRecorder:
```

Comment:
what is the equivalent in gym v1.0?

Reply:
`RecordVideo` wrapper - https://github.com/Farama-Foundation/Gymnasium/blob/f0202ae350c885bb8317858c965312d9459e7155/gymnasium/wrappers/rendering.py#L155